### ISSUE TITLE
Could not find generator 'rails_event_store:migrate.

### ISSUE THREAD

**Issue by Guillaumejfrt** [RailsEventStore/rails_event_store#204]
Hi,

I develop a Rails 5 app. It added your gem in the gemfile `gem 'rails_event_store'` and run `rails generate rails_event_store:migrate` 

I got the issue: 
```
Running via Spring preloader in process 9779
Expected string default value for '--controller'; got false (boolean)
Expected boolean default value for '--markerb'; got :erb (string)
Could not find generator 'rails_event_store:migrate'. Maybe you meant 'active_record:migration', 'rails_event_store_active_record:migration' or 'active_record:devise'
Run `rails generate --help` for more options.
```

I don't know where it comes from. Could you help me?

Thanks! Guillaume

**Comment by paneq** [comment]
Also, the command might be `bin/rails generate rails_event_store_active_record:migration` (where did you find `rails_event_store:migrate` ?)

**Comment by Guillaumejfrt** [comment]
I did `spring stop` before `rails_event_store:migrate` but it did not work.

I found the `rails_event_store:migrate` command at the beginning of this documentation :) : https://blog.arkency.com/2015/04/the-event-store-for-rails-developers/ and ruby doc also: http://www.rubydoc.info/gems/rails_event_store/0.2.0

It finally worked with first `spring stop` and second `bin/rails generate rails_event_store_active_record:migration` 

Thanks @paneq !

**Comment by paneq** [comment]
@Guillaumejfrt thanks for that info. As you can see the article is 2.5 years old and rails_event_store v0.2.0 was released on 0.2.0 - January 29, 2016 (9.5 KB) so that's also quite an old rubydoc documentation:

Please follow http://railseventstore.org/ documentation in the future and https://github.com/RailsEventStore/rails_event_store/releases for new releases and changes between versions.

Also, we would be happy if you fill out our short survey about RES usage: https://goo.gl/forms/ecNpJsL5qM2M8Ls93 if you haven't yet :) Thank you and have a great day!

**Comment by Guillaumejfrt** [comment]
sorry @paneq ! I did not see that but it was the first google result!

Thanks for the links

I fill your survey

All the best
